.storybook/main.js

```diff
@@ -64,8 +67,18 @@ module.exports = {
         include: [modulesPath],
         exclude: /examples|stories|spec|codemod|docs/,
         loaders: [
+          // loaders are run in reverse order. symbol-doc-loader needs to be done first
+          {
+            loader: path.resolve(__dirname, 'style-transform-loader'),
+            options: {
+              Doc,
+            },
+          },
```

Comment:
is this what adds the transform as part of webpack?

Reply:
Yes